src/data/resources/testbeds/testbeds.yaml

```diff
@@ -0,0 +1,54 @@
+
+- name: Chameleon Cloud
+  url: https://chameleoncloud.org/
+  image: ./chameleon-cloud.png
+  description: >
+    Chameleon is a large-scale, deeply reconfigurable experimental platform built to support Computer Sciences systems research.  
+
+- name: CloudLab
+  url: https://www.cloudlab.us/
+  image: ./cloudlab.png
+  description: >
+    Flexible, scientific infrastructure for research on the future of cloud computing. Researchers use CloudLab to build their own clouds, experimenting with new architectures that will form the basis for the next generation of computing platforms. 
+
+- name: Cosmos
+  url: https://powderwireless.net/
```

Comment:
The link should be: https://www.cosmos-lab.org/

Reply:
thanks for catching that!